src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -1050,7 +1050,7 @@
                                 <ColumnDefinition Width="24"/>
                                 <ColumnDefinition Width="*"/>
                             </Grid.ColumnDefinitions>
-                            <fabric:FabricIconControl x:Name="fbIcn" VerticalAlignment="Center" Grid.Column="0" GlyphSize="Small" GlyphName="{Binding Tag, RelativeSource={RelativeSource Mode=TemplatedParent}}"/>
+                            <fabric:FabricIconControl x:Name="fbIcn" VerticalAlignment="Center" Grid.Column="0" GlyphSize="Small" GlyphName="{Binding Tag, RelativeSource={RelativeSource Mode=TemplatedParent}}" ShowInControlView="False"/>
```

Comment:
![copy selected properties menu icon](https://user-images.githubusercontent.com/7775527/64989072-9d1fd280-d881-11e9-8991-6136e3675c1a.png)
![context menu items for expand descendants and filing issues](https://user-images.githubusercontent.com/7775527/64989101-a9a42b00-d881-11e9-9181-af4485563a12.png)
![context menu items for events and filing issues](https://user-images.githubusercontent.com/7775527/64989154-c7719000-d881-11e9-9cfb-7cf0988ed176.png)